cargo-dist/templates/ci/github/partials/publish_homebrew.yml.j2

```diff
@@ -13,13 +13,14 @@
       GITHUB_EMAIL: "admin+bot@axo.dev"
     if: ${{ !fromJson(needs.plan.outputs.val).announcement_is_prerelease || fromJson(needs.plan.outputs.val).publish_prereleases }}
     steps:
-      - uses: actions/checkout@v4
+      - uses: {{{ actions["actions/checkout"] | safe }}}
         with:
+          persist-credentials: false
```

Comment:
This snuck in the wrong branch - looks like it's there upstream too. Will change.